lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -11,52 +12,62 @@ def add_index(table_name, column_name, **options)
       remove_index(table_name, name: index_name)
       super
     end
+    ruby2_keywords :add_index if respond_to?(:ruby2_keywords, true)
 
-    def add_column(table_name, column_name, type, options = {})
+    def add_column(table_name, column_name, type, *)
       return super unless column_exists?(table_name, column_name)
 
       SafePgMigrations.say("/!\\ Column '#{column_name}' already exists in '#{table_name}'. Skipping statement.", true)
     end
+    ruby2_keywords :add_column if respond_to?(:ruby2_keywords, true)
 
-    def remove_column(table_name, column_name, type = nil, options = {})
+    def remove_column(table_name, column_name, type = nil, *)
```

Comment:
Same for `remove_column`. `def remove_column(table_name, column_name, type = nil, **options)` should be enough.